crates/test_utils/tests/connectors/selenium.rs

```diff
@@ -847,7 +877,12 @@ fn get_firefox_profile_path() -> Result<String, WebDriverError> {
             fp.join(&MAIN_SEPARATOR.to_string())
         })
         .unwrap();
-    base_path.push_str(r#"/Library/Application Support/Firefox/Profiles/hs-test"#); //Issue: 1573
+    if env::consts::OS == "macos" {
+        base_path.push_str(r#"/Library/Application Support/Firefox/Profiles/hs-test"#);
+    //Issue: 1573
+    } else if env::consts::OS == "linux" {
+        return Ok(r#"/home/runner/.mozilla/firefox/hs-test"#.to_string());
+    }
```

Comment:
1. You can mark that this PR closes #1573.
2. The path and Firefox profile name is very specific to GitHub Actions runner. Can we substitute `/home/runner` by the value of `${HOME}` environment variable?
3. For setting the name of the profile for GitHub Actions, we can make use of the `CI` or `GITHUB_ACTIONS` environment variables (both will be `true`). Reference: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables